This patch fixes Issue 3

gregorydulin opened the pull request.

livibetter commented:
Three things before I merge:

1. Could you use `[[` instead of `[` since this is for Bash?
2. Also just remove ` #>> ...` no need for that when we are using VCS.
3. Please don't include that merge pull request commit.

I am expecting a single clean commit, thanks. (If you don't know how to clean up, just create a new branch, redo it, and make new PR, we can close this one and use another PR)

gregorydulin commented:
You're right, I don't know how to clean up, so I made a new Pull Request.

I *think* I addressed all your concerns.  If I didn't, I apologize in advance (I'm not terribly familiar with VCS).  Thanks for the pointer on keeping the commits clean.

livibetter commented:
Closing in favor of #5.
